.drone.yml

```diff
@@ -325,15 +364,20 @@ steps:
 
 - name: publish-prod-bintray
   image: casperlabs/node-build-u1804
+  # Removed casper-node from tag publish as we not longer ship this way.
   commands:
-  - "./upload.sh --repo-name debian --package-name casper-node --package-tag true"
```

Comment:
i feel this should be added back unless this uploaded in a different automation step

Reply:
This was published to participate in our networks, which will be delivered via bin.tar.gz now.  Installing the deb will not be compatible.